common/tile.cpp

```diff
@@ -761,28 +763,56 @@ bool tile_apply_activity(struct tile *ptile, Activity_type_id act,
    Add one entry about pollution situation to buffer.
    Return if there has been any pollution (even prior calling this)
  */
-static bool tile_info_pollution(char *buf, int bufsz,
-                                const struct tile *ptile,
-                                struct extra_type *pextra, bool prevp,
+static bool tile_info_pollution(QString &buf, QString pollution, bool prevp,
```

Comment:
Pass by const reference to avoid a copy
```suggestion
static bool tile_info_pollution(QString &buf, const QString &pollution, bool prevp,
```

Reply:
Done